assets/src/components/cluster/pods/PodInfo.tsx

```diff
@@ -27,14 +29,30 @@ export const statusesToRecord = (statuses?: Maybe<Maybe<ContainerStatus>[]>) =>
     {} as Record<string, Maybe<ContainerStatus>>
   )
 
-function getLogUrl({ name, namespace }) {
-  return `/apps/${namespace}/logs?${asQuery({ pod: name })}`
+function useGetLogUrl({
+  name,
+  namespace,
+}: {
+  name?: string
+  namespace?: string
+}) {
+  const isApp = useNamespaceIsApp(namespace)
```

Comment:
could we just pass this as an argument and ignore the potentially risky app name fetch?

Reply:
Is there some field on a Pod that I could use to determine if it's an app or not? Otherwise I'd just be pushing the app name fetch further up the render tree.

Reply:
Not easily, I thought you might be able to determine it from context of where the component is called, eg if it's called from an `/apps/` prefixed url use something like that link, otherwise non-app link?

Reply:
Nah, I need to know whether it's an app or not on the Pod page, where it's not obvious.

Reply:
Either that, or we always show the logs on the Pod page, even if there is an associated App page where you could also view the logs.